source/edge-server/configure.txt

```diff
@@ -10,4 +10,307 @@ Configure Edge Server
+to the general internet. 
+
+.. _edge-server-os-support:
+
+OS Compatibility
+~~~~~~~~~~~~~~~~
+
+The Edge Server has been verified with the following operating systems:
+
+- Ubuntu Server 18+
+- macOS 11.x and newer running on M1 or Intel 2018+ MacBook Pros
+
+Other operating systems may be possible, but have not yet been verified.
+
+.. _edge-server-hardware-requirements:
+
+Hardware Requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+Edge Server benchmarking is not yet complete, so minimum required hardware
+is not yet available. 
+
+.. _edge-server-install-and-configure:
+
+Install and Configure the Edge Server
+-------------------------------------
+
+Set up and run the Edge Server on the host.
+
+.. include:: /includes/important-stop-tiered-sync-server.rst
+
+.. procedure::
+
+   .. step:: Get the Edge Server Code
+
+      Use ``wget`` to get the current Edge Server code as a ``.tar`` file:
+
+      .. code-block:: shell
+
+         wget --content-disposition https://realm.mongodb.com/api/client/v2.0/tiered-sync/package/latest
+
+      Unzip the tar to get the files.
+
+      .. code-block:: shell
+
+         tar -xvf *.tgz
+
+      You now have an ``edge_server`` directory containing the server files.
+
+   .. step:: Complete the Edge Server Configuration Details
+
+      The ``edge_server`` directory contains a :file:`config.json` file you 
+      must edit with the appropriate values to configure the server.
+
+      .. code-block:: json
+
+         {
+           "clientAppId": "YOUR-APP-ID",
+           "query": {
+             "YOUR-COLLECTION-NAME": "YOUR-QUERY"
+           },
+           "cloudSyncServerAuthSecret": "YOUR-AUTH-SECRET",
+           "hostname": "YOUR-EDGE-SERVER-DOMAIN"
+         }
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Field name
+           - Value
+
+         * - ``clientAppId``
+           - The App ID of the App Services App you created above.
+
+         * - ``query``
+           - An object containing one or more collections and queries that 
+             define the collections and documents the Edge Server can 
+             sync.
+
+         * - ``YOUR-COLLECTION-NAME``
+           - The name of the collection in your database where the Edge 
+             Server should sync data. If you're using a 
+             template app, this is the ``Item`` collection.
+            
+         * - ``YOUR-QUERY``
+           - This query determines what documents the Edge Server
+             can sync with the collection. For example, ``truepredicate``
+             means the Edge Server syncs all documents with the collection.
+             You can use :ref:`Realm Query Language <rql>` to define this
+             query, with the same caveats around :ref:`Flexible Sync RQL 
+             Limitations <flexible-sync-rql-limitations>`.
+
+         * - ``cloudSyncServerAuthSecret``
+           - Give your :ref:`App Services App ID <find-app-id>` to your 
```

Comment:
```suggestion
           - The authorization secret that your Product or Account Representative generated for your App. 
```
only for consistency on these definitions. feels weird to have an instruction embedded in here, esp if it's the first time it's mentioned